src/Lower.cpp

```diff
@@ -220,6 +221,14 @@ Stmt lower(const vector<Function> &outputs, const string &pipeline_name, const T
     s = simplify(s);
     debug(2) << "Lowering after partitioning loops:\n" << s << "\n\n";
 
+    if (t.has_feature(Target::HVX_128) ||
```

Comment:
This could be

    if (t.features_any_of({Target::HVX_64, Target::HVX_128}))

(That's what I've been doing all over the place for similar code.)